packages/page-accounts/src/Accounts/BannerExtension.tsx

```diff
@@ -63,31 +62,23 @@ function BannerExtension (): React.ReactElement | null {
 
   return (
     <Banner type='warning'>
-      <p>{t<string>('It is recommended that you create/store your accounts securely and externally from the app. On {{yourBrowser}} the following browser extensions are available for use -', {
-        replace: {
-          yourBrowser: stringUpperFirst(browserName)
-        }
-      })}</p>
-      <ul>{availableExtensions[browserName].map(({ desc, link, name }): React.ReactNode => (
-        <li key={name}>
-          <a
-            href={link}
-            rel='noopener noreferrer'
-            target='_blank'
-          >
-            {name}
-          </a> ({t(desc)})
-        </li>
-      ))
-      }</ul>
-      <p>{t<string>('Accounts injected from any of these extensions will appear in this application and be available for use. The above list is updated as more extensions with external signing capability become available.')}&nbsp;
+      <p><b>Action needed: Move you accounts to a browser extension</b></p>
+      <p>
+        {t<string>('Managing accounts directly via {{host}} is being phased out, to encourage more', { replace: { host: window.location.host } })}
+        {t<string>('secure ways to manage your keys. We recommend, that you use one of the supported')}
+        {t<string>('browser extensions. You can learn')}&nbsp;
```

Comment:
there is a space missing between `supported` and `browser` :sweat_smile:

Reply:
added